src/ray/common/ray_syncer/ray_syncer_bidi_reactor.h

```diff
@@ -93,11 +94,24 @@ class RaySyncerBidiReactor {
     }
   };
 
+  /// Set rpc completion callback, which is called after rpc read finishes.
+  /// This function is expected to call only once, repeated invocations throws exception.
```

Comment:
```suggestion
  /// This function is expected to call only once, repeated invocations will check fail.
```

Reply:
I'm not sure why "will check fail" is a better wording?

Reply:
Anyway I updated the wording, but would like to hear the reasoning.